content/projects/tdd/string-calculator-part-1/_index.md

```diff
@@ -166,13 +166,13 @@ In the first case, the default delimiter is: ";".
 
 In the second case, the default delimiter is: "4".
 
-**Note:** you are not expected to handle integer delimiters where the delimiter and the digit you are adding are identical and next to each other. Example:
+**Note:** You are not expected to handle integer delimiters where the single digit at the start and/or end of the delimiter and the preceding or subsequent integer you are adding are identical. Example:
```

Comment:
@Lindo-code still not quite right...

This input_string "//82\n182282" meets the criteria you are describing but should be handled...
  - 2 is the integer at the end of the delimiter and the also the preceding integer.